docs/reference/modules/discovery/adding-removing-nodes.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[modules-discovery-adding-removing-nodes]]
+=== Adding and removing nodes
+
```

Comment:
It seems like the majority of this information pertains only to removing nodes.   It's not necessary immediately, but at some point I think it would be good to split this into two separate pages -- one about adding nodes (with lots of details about how that differs depending on platform and node type, etc) and one about removing nodes (which would be most of this content).